src/styles.scss

```diff
@@ -1,3 +1,22 @@
+$pink-color: #e10072 !default;
+$blue-light: #0094df !default;
```

Comment:
In any case I would also add a default `blue` color, so there's a base reference.

Reply:
$blue-light was changed to the $blue-color